README.md

```diff
@@ -44,15 +44,30 @@ Everyone is asked to follow our [code of conduct](CODE_OF_CONDUCT.md) and to che
 
 If you are not familiar or confident contributing on GitHub, you can also contribute a case study and your tips and tricks via our [Google submission form](https://goo.gl/forms/akFqZEIy2kxAjfZW2). 
 
+### Citing _The Turing Way_
+
+You can reference _The Turing Way_ through the project's Zenodo archive using doi: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
+DOIs allow us to archive the repository and they are really valuable to ensure that the work is tracked in academic publications.
```

Comment:
I'm not sure if there's a style guide but above we have "doi" and "DOI". I prefer DOI myself.

Reply:
Fixed! Thank you!